.github/workflows/sync-upstream.yml

```
@@ -0,0 +1,158 @@
+      run: |
+        git push --force origin "upstream/${UPSTREAM_BRANCH}:refs/heads/${SYNC_BRANCH}"
+
+        pr_number=$(gh pr list \
+          --head "$SYNC_BRANCH" \
+          --base main \
+          --state open \
+          --json number \
+          --jq '.[0].number // empty')
+
+        body=$(cat <<'EOF'
+Automated upstream sync found merge conflicts with the fork's custom changes.
```

actionlint (1.7.12)

[error] could not parse as YAML: could not find expected ':'

(syntax-check)